DevelopedCalculator.js

```diff
@@ -9,9 +10,13 @@ class DevelopedCalculator extends Calculator {
         this.hostElem.prepend(historyDiv);
 
 
-        this.updateResultElement();
+        const currentResultUpdate = []
+    }
 
+    onButtonNumberClick_(event) {
```

Comment:
Can you tell me why the method name ends with _ ?

Reply:
then create onButtonNumberClick on the new class
it's very important that the name of the method matches with the method on the parent class

Reply:
so did you follow what I said "the name of the method matches with the method on the parent class"